src/bin/ion/commands/beta/generate/generator.rs

```diff
@@ -67,12 +71,13 @@ impl<'a> CodeGenerator<'a, RustLanguage> {
 
 impl<'a> CodeGenerator<'a, JavaLanguage> {
     pub fn new(output: &'a Path, namespace: &'a str) -> CodeGenerator<'a, JavaLanguage> {
-        let tera = Tera::new(&format!(
-            "{}/src/bin/ion/commands/beta/generate/templates/java/*.templ",
-            env!("CARGO_MANIFEST_DIR")
-        ))
-        .unwrap();
-
+        let mut tera = Tera::default();
+        // Add all templates using `java_templates` module constants
+        // This allows packaging binary without the need of template resources.
+        let _ = tera.add_raw_templates(vec![
+            ("class.templ", templates::java::CLASS),
+            ("nested_type.templ", templates::java::NESTED_TYPE),
+        ]);
```

Comment:
Rather than ignoring the result, we should either use `?` or `unwrap()` or something similar. 
```suggestion
        tera.add_raw_templates(vec![
            ("class.templ", templates::java::CLASS),
            ("nested_type.templ", templates::java::NESTED_TYPE),
        ]).unwrap();
```